packages/shared/src/api/survey/surveyPostRequest.ts

```diff
@@ -0,0 +1,20 @@
+export interface SurveyPostRequest {
+  fullName: string;
+  email: string;
+  town: string;
+  birthYear: number;
+  gender: string;
+  educationStatus: string;
+  courseInformationSource: string[];
+  associatedWords: string[];
+  description: string;
+  prevParticipation: string;
+  reasonForRetakingCourse: string | null;
```

Comment:
Null or undefine? @KonradSzwarc what are best practices in TS? I'm in favor or undefined and avoid null whenever possible.